lib/zold/commands/alias.rb

```diff
@@ -27,10 +27,34 @@ def run(args = [])
       mine = Args.new(opts, @log).take || return
       command = mine[0]
       raise "A command is required, try 'zold alias --help'" unless command
+      case command
+      when 'set'    then set(mine[1], mine[2])
```

Comment:
@botanicus please don't make us guess what `mine[1]` is, extract variables.

Reply:
@ledestin it's how it's been used in other commands as well, for instance https://github.com/zold-io/zold/blob/master/lib/zold/commands/remote.rb#L120

If you think this is wrong, then it should be changed everywhere, not just here, in which case I think bug report is where it belongs.

I personally don't think there's anything wrong with it since they are named in both method signature and in the help just above it.

I'm not sure what @yegor256 thinks, I've seen something from him about avoiding unnecessary variables, but I cannot tell if this is one of the cases or not.

Anyhow ... since it's not specific to this code, I'd suggest a bug report. Makes sense?

Reply:
@botanicus sure!